product_barcode_generator/models/product.py

```diff
@@ -36,21 +36,20 @@ class ProductCategory(models.Model):
 class ProductProduct(models.Model):
     _inherit = 'product.product'
 
-    ean13 = fields.Char(copy=False)
     ean_sequence_id = fields.Many2one('ir.sequence', string='Ean sequence')
```

Comment:
I still believe that OCA shall speed PR merging up someway. One of ideas is creating new issues instead of waiting until every concern is resolved